STREAMS-1279 - Describe webhook exchanges API. (#56)
Axway/streams-open-docs: content/en/docs/subscribers/subscriber-webhook.md

| 200 Ok | Indicates that the subscription requested is valid and has been retrieved. |
| 404 Not found | Indicates that the requested URL or subscription requested does not exist. |

## Getting webhook exchanges for a subscription

> In order to know all exchanges done with the `webhookUrl` of a subscription during the last 5 minutes, simply do the following GET request:

Reviewer: I will say 'all exchanges done for a subscription'

Author: Updated!

`GET /subscribers/webhook/subscriptions/{subscriptionId}/exchanges`

> The default time window of the retrieved history can be changed with `start` and `end` query params (date-time in ISO 8601 format, eg: 2021-01-10T10:13:32Z).

Reviewer: Provide an full example as above for default value

Author: A second example with full parameters has been added.

| 200 Ok | Indicates that the subscription requested is valid and has been retrieved. |
| 404 Not found | Indicates that the requested URL or subscription requested does not exist. |

> ## Getting webhook exchanges for a subscription

Reviewer 2: ```suggestion
## Getting the webhook notification history for a subscription
```

> In order to know all exchanges done for a subscription during the last 5 minutes, simply do the following GET request:

Reviewer 2: ```suggestion
In order to retrieve the history of webhook exchanges (requests and responses) that have occurred for a subscription in the last 5 minutes, just do the following `GET` request:
```

> In order to retrieve the history of webhook exchanges (requests and responses) that have occurred for a subscription in the last 5 minutes, just do the following `GET` request:

Reviewer 3: ```suggestion
Use the following `GET` request to retrieve the history of webhook exchanges (requests and responses) that have occurred for a subscription in the last 5 minutes:
```

`GET /subscribers/webhook/subscriptions/{subscriptionId}/exchanges`

> The default time window of the retrieved history can be changed with `start` and `end` query params (date-time in ISO 8601 format, eg: 2021-01-10T10:13:32Z):

Reviewer 3: ```suggestion
You can change the default time window of the retrieved history with `start` and `end` query params (date-time in ISO 8601 format, eg: 2021-01-10T10:13:32Z):
```

`GET /subscribers/webhook/subscriptions/{subscriptionId}/exchanges?start=2021-01-10T10:13:30Z&end=2021-01-10T10:13:32Z`

### Webhook exchanges status codes

> Below the list of HTTP status codes that can be returned when trying to get exchanges history:

Reviewer 2: ```suggestion
Below the list of HTTP status codes that can be returned when interacting with the exchanges history:
```

> Below the list of HTTP status codes that can be returned when interacting with the exchanges history:

Reviewer 3: ```suggestion
The following HTTP status codes can be returned when interacting with the exchanges history:
```

| Code | Comment |
|------|---------|
| 200 Ok | Indicates that the exchanges history requested is valid and has been retrieved. |

> | 400 Bad Request | Indicates that the provided data are invalid. |

Reviewer 2: ```suggestion
| 400 Bad Request | Indicates that the request is invalid. |
```
